### DISCUSSION TITLE
Implementing msg/call on high/low alarms

### DISCUSSION THREAD

**Discussion by ibrodezk** [NightscoutFoundation/xDrip#3109] [Ideas]
I am a programmer and a newly diagnosed T1D and wanted to ask if anyone thought of a feature to add to alarms.
On extreme values send a message/call a saved number ( a family member).
Wanted to know if anyone thought of it and if there are any objections to me implementing this.

**Comment by Navid200** [comment]
Please have a look at this first: https://navid200.github.io/xDrip/docs/Emergency.html

Please tell us if there is anything you want that does not already cover.
